runtime/account_keys_test.go

```diff
@@ -966,3 +967,95 @@ type testAccountKeyStorage struct {
 	keys        []*AccountKey
 	returnedKey *AccountKey
 }
+
+func TestPublicKey(t *testing.T) {
+
+	t.Parallel()
+
+	rt := NewInterpreterRuntime()
+
+	runtimeInterface := &testRuntimeInterface{
+		validatePublicKey: func(publicKey *PublicKey) (bool, error) {
+			if len(publicKey.PublicKey) == 0 {
```

Comment:
Maybe assert here that the function is called, by declaring a variable outside, setting it to true here, and asserting outside it was true after the execution

Reply:
That's a good idea. Added in 03fa1ce